### DISCUSSION TITLE
Nginx is not working

### DISCUSSION THREAD

**Discussion by Kurra432** [learndevopsonline/learndevopsonline#1526] [Q&A]
Hi 

I got the same issue after giving the private IP's in frontend I am getting error.
you previously told me delete the nginx and create new one, I was created  new one it's working but again giving any private IP's its not working 
![Screenshot frontend-10-13 112829](https://github.com/learndevopsonline/learndevopsonline/assets/129031517/653bfc94-4bb0-4202-aed8-334e355b329f)

**Comment by sreedharm07** [comment]
Hi @Kurra432 
give double slash here
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ab536f35-d62f-4a74-ba65-20b58d8d16d6)

**Reply by Kurra432** [reply]
ok bro thanks
